packages/app-cli/tests/services/plugins/api/JoplinSettings.ts

```diff
@@ -66,4 +67,38 @@ describe('JoplinSettings', () => {
 		await service.destroy();
 	});
 
+	test('should allow registering multiple settings', async () => {
+		const service = new newPluginService() as PluginService;
+
+		const pluginScript = newPluginScript(`
+			joplin.plugins.register({
+				onStart: async function() {
+					await joplin.settings.registerSettings({
+						'myCustomSetting1': {
+							value: 1,
+							type: 1,
+							public: true,
+							label: 'My Custom Setting 1',
+						},
+						'myCustomSetting2': {
+							value: 2,
+							type: 1,
+							public: true,
+							label: 'My Custom Setting 2',
+						}
+					})
+				},
+			});
+		`);
+		const plugin = await service.loadPluginFromJsBundle('', pluginScript);
+		await service.runPlugin(plugin);
+
+		await expectNotThrow(() => Setting.value('plugin-org.joplinapp.plugins.PluginTest.myCustomSetting1'));
+		await expectNotThrow(() => Setting.value('plugin-org.joplinapp.plugins.PluginTest.myCustomSetting2'));
```

Comment:
remove these two lines please, as the lines below already test that the call doesn't fail.